src/capture.cpp

```diff
@@ -1098,7 +1110,11 @@ void acquisition::Capture::run_soft_trig() {
                 ROS_INFO_STREAM(" Recorded frames "<<count<<" / "<<nframes_);
                 if (count > nframes_) {
                     ROS_INFO_STREAM(nframes_ << " frames recorded. Terminating...");
+#if (OPENCV_VERSION < 4)
+                    cvDestroyAllWindows();
```

Comment:
I think this is flipped.

Also imwite on line 835 needs this condition i think.c

Reply:
And imshow on 677.

Reply:
And display overlay on line 1049.